proto/confio/v1/proofs.proto

```diff
@@ -1,16 +1,18 @@
 syntax = "proto3";
 
-package ics23;
+package confio.v1.ics23;
```

Comment:
this will need to be regenerated, want to double check if its okay before doing so. 

DO NOT MERGE

Reply:
Sure, we can rename this.
Let's make sure this gets into a 0.7.0 release as it is breaking for the client.
I assume the wire format doesn't change, which we do try to avoid.

Reply:
correct wire format does not change. Would you be up for cutting 1.0?